dependency-path-mapping.php

```diff
@@ -70,7 +71,14 @@ function dependency_src_to_fs_path( $src ) {
 		if ( empty( $src_parts['host'] ) ) {
 			// With no host, this is a path relative to the WordPress root
 			$fs_path = "{$this->site_dir}{$path}";
-			return file_exists( $fs_path ) ? $fs_path : false;
+			if ( file_exists( $fs_path ) ) {
+				// Get rid of duplicate slashes without resolving symlinks.
+				// We need to avoid resolving symlinks to be able to provide actual
+				// and relative FS paths to the concat script.
+				// TODO: Review and improve this explanation.
+				$fs_path = dirname( $fs_path ) . '/' . basename( $fs_path );
```

Comment:
This is not a return, but it should be.

Reply:
Thanks for pointing that out. I shouldn't have let that change land since it wasn't covered by the new tests. I'll fix in a follow-up PR with a test.